bindings/c/test/mako/utils.c

```diff
@@ -80,6 +80,18 @@ void genkey(char* str, char* prefix, int prefixlen, int prefixpadding, int num,
 	str[len - 1] = '\0';
 }
 
+void updateKeyPrefix(char* str,int prefixlen, int prefixpadding, int len){
+	//Acquire a random prefix
+	const int prefixoffset = prefixpadding ? len - prefixlen - 1 : 0;
+	char* prefixstr = (char*)malloc(sizeof(char)*prefixlen+1);
+	randstr(prefixstr,prefixlen+1);
```

Comment:
Let's keep the first 4 characters as "mako", and randomize the next `len-4` characters with `randnumstr`

Reply:
e.g.
If we want to make the prefix 8 bytes, generate something like this:
mako0123

Then, in the key generation, use this as the prefix.
e.g.
mako01235943728
mako01230184234
mako01235431637
...

Reply:
Thanks! It's way more clear now.